paymentsheet/src/main/java/com/stripe/android/paymentsheet/addresselement/AddressElementActivity.kt

```diff
@@ -82,23 +81,14 @@ internal class AddressElementActivity : ComponentActivity() {
 
             LaunchedEffect(Unit) {
                 modalBottomSheetState.show()
-            }
-
-            LaunchedEffect(Unit) {
-                // We need to drop(1) to avoid the sheet being closed on the first composition,
-                // given that the initial bottom sheet state is `hidden`.
-                snapshotFlow { modalBottomSheetState.currentValue }.drop(1).collect {
-                    // finish the activity when the sheet closes.
-                    if (it == ModalBottomSheetValue.Hidden) {
-                        finish()
-                    }
-                }
```

Comment:
Seems like we don’t actually need this. Since `modalBottomSheetState.hide()` is suspending, we can just call `finish()` after `hide()` completes.